.github/workflows/golangci-lint.yaml

```diff
@@ -27,9 +27,9 @@ jobs:
           go-version: 'stable'
 
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v4
+        uses: golangci/golangci-lint-action@v6
         with:
-          version: v1.56
+          version: v1.60.2
```

Comment:
Can you confirm that https://github.com/redpanda-data/common-go/actions/runs/10511516964/job/29122660139 is not related to this change?

Reply:
Issue started prior to this PR: 

https://github.com/redpanda-data/common-go/actions/runs/10394862238/job/28835193918